assets/samplesheet.csv

```diff
@@ -1,3 +1,3 @@
 sample,fastq_1,fastq_2
-SAMPLE_PAIRED_END,/path/to/fastq/files/AEG588A1_S1_L002_R1_001.fastq.gz,/path/to/fastq/files/AEG588A1_S1_L002_R2_001.fastq.gz
-SAMPLE_SINGLE_END,/path/to/fastq/files/AEG588A4_S4_L003_R1_001.fastq.gz,
+test,https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/reads_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/reads_2.fastq.gz
+novirusreads,https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/novirus_reads_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/novirus_reads_2.fastq.gz
```

Comment:
```suggestion
novirusreads,https://github.com/nf-core/test-datasets/raw/viralintegration/testdata/novirus_R1.fastq.gz,https://github.com/nf-core/test-datasets/raw/viralintegration/testdata/novirus_R2.fastq.gz
```

Reply:
Oh, I see now that the filenames have changed, that's why it 404s :)

Reply:
Whoops! I updated the sample sheet in the test-dataset repo. I personally don't like that because I feel like the test sample sheet and the asset sample sheet always get out of sync. I like the way sarek does it:
https://github.com/nf-core/sarek/blob/f31ae00d2fa1f411d6b3daae3faf8ce310cd48fc/conf/test.config#L23

At-least for early development until things stabilize. Any thoughts?

Reply:
That sounds like a good plan to me!